src/components/global/TaskList.jsx

```diff
@@ -13,6 +15,12 @@ function TaskList(props) {
   const { onTaskClick, tasks, onNewTask } = props;
   const [currentTask] = useContext(CurrentTaskContext);
   const { isDarkMode } = React.useContext(DarkModeContext);
+  const { currentUser } = useContext(AuthContext);
+  const [subtasks, setSubtasks] = useState([]);
+
+  useEffect(() => {
+    fetchSubtasks(currentUser.uid).then((res) => setSubtasks(res));
```

Comment:
I think this should be 1 level higher in the pages so that you can refetch the subtasks once one has been added so it will appear on the screen?

Reply:
that's a good point, will make it so refetch happens correctly